janusgraph-es/src/test/java/org/janusgraph/diskstorage/es/ElasticsearchIndexTest.java

```diff
@@ -323,6 +323,50 @@ public void testCustomMappingProperty() throws BackendException, IOException, Pa
         IOUtils.closeQuietly(response);
     }
 
+    /**
+     * Helper function used by testSetCardinality and testListCardinality.
+     * Performs initialization and assertion checks that are common for both tests.
+     */
+    private void testCollectionCardinality(String collectionName) throws Exception {
```

Comment:
Not critical but could you please move this method below `testListCardinality` method?

Reply:
Removed this method in favour of parametrized test as suggested above.